src/Compilers/CSharp/Portable/Binder/MethodGroupResolution.cs

```diff
@@ -88,7 +88,7 @@ public bool IsExtensionMethodGroup
         }
 
         public bool IsLocalFunctionInvocation =>
-            MethodGroup.Methods.Count == 1 && // Local functions cannot be overloaded
+            MethodGroup?.Methods.Count == 1 && // Local functions cannot be overloaded
```

Comment:
This is another little bug fix while I was in the area. It prevents the debugger from displaying exceptions for this property when viewing the members of a `MethodGroupResolution`.

Reply:
No, because there is no crash. It is just very poor experience in the debugger that I am fixing.